frontend/src/data/variables/BrfssProvider.ts

```diff
@@ -98,8 +96,26 @@ class BrfssProvider extends VariableProvider {
           estimated_total_diabetes: (series) => series.sum(),
         })
         .resetIndex();
+
+      // We need to get the national acs dataset here in order to
+      // get the national pct share of population for each state.
+      acsBreakdowns.geography = "national";
```

Comment:
Would be good to add a TODO here to see if we can convert these two calls to ACS provider into a single call

Reply:
Or actually, we want to do this on the BE eventually, so adding that as the TODO would be better

Reply:
Yeah I'll add that, thanks